### DISCUSSION TITLE
aiscatcher.org dashboard

### DISCUSSION THREAD

**Discussion by otm64** [jvde-github/AIS-catcher#199] [Q&A] [answered]
@jvde-github Not really sure what is the best way to reach you about adding my site to your dashboard so I will go this route. Let me know the next step(s).

Mike

**Comment by jvde-github** [comment, marked as answer]
> @jvde-github Not really sure what is the best way to reach you about adding my site to your dashboard so I will go this route. Let me know the next step(s).
> 
> 
> 
> Mike

Hi, can you drop me a note on jvde.github@gmail.com? I will set you up, thanks!

**Reply by otm64** [reply]
Ok

**Reply by jvde-github** [reply]
https://aiscatcher.org/tyreso 

Thanks for your support.
